src/js/common/schemaform/FormSaved.jsx

```diff
@@ -7,9 +7,19 @@ import Scroll from 'react-scroll';
 
 import { focusElement } from '../utils/helpers';
 import { fetchInProgressForm, removeInProgressForm } from './save-load-actions';
+import { handleVerify } from '../../common/helpers/login-helpers.js';
 
 import FormStartControls from './FormStartControls';
 
+moment.updateLocale('en', {
```

Comment:
I'm not exactly sure, it does look like we have both in places. I don't see any uses of periods, though, so I'm hoping we can avoid making this `updateLocale` call and just use the moment formatting (which can do upper or lower case).